OpenActive.Server.NET/StoreBookingEngine/Settings/StoreBookingEngineSettings.cs

```diff
@@ -18,5 +18,6 @@ public class StoreBookingEngineSettings
         public IOrderStore OrderStore { get; set; }
         public bool BusinessToConsumerTaxCalculation { get; set; }
         public bool BusinessToBusinessTaxCalculation { get; set; }
+        public bool UseAsyncWithinOrderTransactions { get; set; }
```

Comment:
Should this be inverted to something like:

`public bool EnforceSyncWithinOrderTransactions { get; set; }`

The reason being that with `EnforceSyncWithinOrderTransactions = true` the `CheckSyncValueTaskWorkedAndReturnResult()` would check to ensure the transactions were synchronous (and be a minor performance improvement); however with it set to `false` it's possible that any store implementations could still be all async or only part async - simply based on whether the body of the method itself includes any `async` calls.

This is probably a feature rather than a bug, as perhaps some implementations will want to be part-async, part-sync. It's really only the fully-sync path that is likely to want to be guaranteed.